.gitlab/ci/global.yml

```diff
@@ -185,6 +185,8 @@
   extends:
     - .default-cache
     - .default-before-script
+  after_script:
+    - echo "done" > ${PIPELINE_JOBS_FOLDER}/${CI_JOB_NAME}.txt
```

Comment:
If you only need "`done`" in that file,
you could accomplish the same with just `touch`ing in file, and check that it exists.